inc/Engine/WPRocketUninstall.php

```diff
@@ -239,6 +250,29 @@ private function delete_config_files() {
 		$this->delete( $this->config_path );
 	}
 
+	/**
+	 * Drop preload tables.
+	 */
+	private function delete_preload_table() {
+		// If the table exist, then drop the table.
+		$this->drop_rucss_current_site_tables();
```

Comment:
```suggestion
		$this->drop_rucss_current_site_tables();
```
Needs to be changed for a method specific to preload tables